migrating/migrating-gateways/ossm-migrating-gateways-assembly.adoc

```diff
@@ -1,8 +1,39 @@
 :_mod-docs-content-type: ASSEMBLY
 [id=ossm-migrating-gateways-assembly]
-= Migrating gateways to Service Mesh 3.0
+= Migrating gateways from Service Mesh 2 to Service Mesh 3
 include::_attributes/common-attributes.adoc[]
 :context: ossm-migrating-gateways-assembly
 
 toc::[]
 
+If you are using gateways with {SMProductName} {SMv2Version} and migrating them to {smproduct} 3.0, you can migrate gateways from {SMProduct} 2 and {SMProduct} 3 just like regular workloads.
+
+Use one of the following two methods for migrating gateways from {SMProductName} 2 to {SMProduct} 3:
+
+* Gateway canary migration
+* Gateway in place migration
+
+include::modules/ossm-migrating-gateways-canary.adoc[leveloffset=+1]
+
+.Next steps
+
+* xref:../done/ossm-migrating-complete-assembly.adoc[Completing your migration]
+
+include::modules/ossm-migrating-gateways-in-place.adoc[leveloffset=+1]
+
+.Next steps
+
+* xref:../done/ossm-migrating-complete-assembly.adoc[Completing your migration]
+
+[role="_additional-resources"]
+[id="addition-resources-gateways_{context}"]
+== Additional Resources
+
+* xref:../multitenant/ossm-migrating-multitenant-assembly.adoc[Multitenant migration guide]
+* Cluster-wide migration guide
```

Comment:
If this won't have an xref until a later effort then I would just comment it out for now:
```suggestion
// * Cluster-wide migration guide
```